pythonequipmentdrivers/oscilloscope/Lecroy_WR8xxx.py

```diff
@@ -478,6 +478,21 @@ def get_trigger_level(self, **kwargs) -> float:
 
         return float(response.lstrip(read_cmd).split()[0])
 
+    def set_trigger_level_auto(self) -> None:
+        """
+        set_trigger_level_auto(**kwargs)
```

Comment:
If the function does not take kwargs as inputs, don't put them in the function prototype within the docstring. Also, since there are no args/kwargs you do not need to list them as None, just omit lines 488-491